src/bia_tischi/__init__.py

```diff
@@ -1,10 +1,9 @@
 __version__ = "0.1.0"
 
 __all__ = (
-    )
+)
 
-from ._machinery import tischi, init_assistant
+from ._machinery import ai, init_assistant
```

Comment:
We should not call it `ai` as others have done that already: https://github.com/jupyterlab/jupyter-ai
I created an issue to discuss naming: https://github.com/haesleinhuepf/bia-tischi/issues/20